battery_management_systems/vitobloc/README.md

```diff
@@ -0,0 +1,21 @@
+# Viessmann Vitobloc 200
+
+This Enapter Device Blueprint integrates Viessmann Vitobloc 200 - combined heat and power unit for natural gas and LPG operation via Modbus implemented on [Enapter Virtual UCM](https://go.enapter.com/handbook-vucm).
+
+## Connect to Enapter
+
+- Sign up to the Enapter Cloud using the Web or mobile app (iOS, Android).
+- Create account to access Viessmann API [here](https://documentation.viessmann.com)
+- Use [Enapter Gateway](https://go.enapter.com/handbook-gateway-setup) to run Virtual UCM.
+- Create [Enapter Virtual UCM](https://go.enapter.com/handbook-vucm).
+- [Upload](https://go.enapter.com/developers-upload-blueprint) this blueprint to Enapter Virtual UCM.
+- Use the `Set Up Connection` command in the Enapter mobile or Web app to set up the Viessmann Vitobloc 200 communication parameters:
+  - User name;
+  - Password.
+
+
+## References
+- [Viessmann Vitobloc 200 product page](https://www.viessmann.de/de/produkte.html)
```

Comment:
maybe better to add link with whole [Vitobloc 200 series ](https://www.viessmann.co.uk/en/products/chp/vitobloc-range.html#models)